sonar-plugin-api/src/main/java/org/sonar/api/batch/fs/InputDir.java

```diff
@@ -26,7 +26,9 @@
  * Layer over {@link java.io.File} for directories. You can access InputDir using {@link FileSystem#inputDir(File)}.
  *
  * @since 4.5
+ * @deprecated since 6.6 Ability to report issues or measures on directories will soon be dropped. Report issues on project if needed. 
  */
+@Deprecated
 public interface InputDir extends InputPath {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)